include/mxnet/kvstore.h

```diff
@@ -223,6 +223,18 @@ class KVStore {
   }
 
   /*!
+   * \return the number of dead node(s) specified by {node_id}
+   * \param node_id can be a node group or a single node
+   * \param timeout a node fails to send heartbeart in {timeout} seconds
+   *        will be presumed as 'dead'
+   *
+   * Always return 0 when type == "local"
+   */
+  virtual int get_dead_num(int node_id, int timeout = 60) const {
```

Comment:
please consider change to `get _num_dead_node` or `get_num_timeout_node`. `dead_num` doesn't sound like english